mne/beamformer/resolution_matrix.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Compute resolution matrix for beamformers."""
+# Authors: olaf.hauk@mrc-cbu.cam.ac.uk
+#
+# License: BSD (3-clause)
+import numpy as np
+
+from ..io.pick import pick_channels, pick_info, pick_channels_forward
+from ..evoked import EvokedArray
+from ..utils import logger
+from ._lcmv import apply_lcmv
+
+
+def make_lcmv_resolution_matrix(filters, forward, info):
+    """Compute resolution matrix for LCMV beamformer.
+
+    Parameters
+    ----------
+    filters : instance of Beamformer
+         Dictionary containing filter weights from LCMV beamformer
+         (see mne.beamformer.make_lcmv).
+    forward : dict
+        Forward Solution with leadfield matrix.
+    info : instance of Info
+        Measurement info used to compute LCMV filters.
+
+    Returns
+    -------
+    resmat : array, shape (n_dipoles_lcmv, n_dipoles_fwd)
+        Resolution matrix (filter matrix multiplied to leadfield from
+        forward solution). Numbers of rows (n_dipoles_lcmv) and columns
+        (n_dipoles_fwd) may differ depending on orientation constraints of
+        filter and forward solution, respectively.
+    """
+    # don't include bad channels from noise covariance matrix
+    bads_filt = filters['noise_cov']['bads']
+    ch_names = filters['noise_cov']['names']
+
+    # good channels
+    ch_names = [c for c in ch_names if (c not in bads_filt)]
+
+    # adjust channels in forward solution
+    forward = pick_channels_forward(forward, ch_names, ordered=True)
+
+    # get leadfield matrix from forward solution
+    leadfield = forward['sol']['data']
+
+    # get the filter weights for beamformer as matrix
+    filtmat = _get_matrix_from_lcmv(filters, forward, info)
+
+    # compute resolution matrix
+    resmat = filtmat.dot(leadfield)
+
+    shape = resmat.shape
+
+    logger.info('Dimensions of LCMV resolution matrix: %d by %d.' % shape)
+
+    return resmat
+
+
+def _get_matrix_from_lcmv(filters, forward, info, max_ori_out='signed',
+                          verbose=None):
+    """Get inverse matrix for LCMV beamformer.
+
+    Parameters
+    ----------
+    filters : instance of Beamformer
+        LCMV spatial filter.
+    forward : dict
+        The forward solution.
+    info : instance of Info
+        Measurement info used to compute the LCMV filters.
+    max_ori_out : ‘signed’
```

Comment:
? maybe you mean str ? or you wnat to list all possible optoins?

Reply:
I copied this from apply_lcmv(), as it's the same max_ori_out parameter.